tests/dash/conftest.py

```diff
@@ -140,7 +155,10 @@ def _apply_config(config_info):
         template_name = "{}.j2".format(config)
         dest_path = "/tmp/{}.json".format(config)
         render_template_to_host(template_name, duthost, dest_path, config_info, op=op)
-        apply_swssconfig_file(duthost, dest_path)
+        if ENABLE_GNMI_API is True:
```

Comment:
It is better to change to `if ENABLE_GNMI_API`

Reply:
updated